lab-ben/test/reader-test.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const expect = require('chai').expect;
+const reader = require('./../lib/reader.js');
+// let correct = [ '4d616563656e6173', '5065616e75744275', '5468697320697320' ];
```

Comment:
No points lost for this, but make sure to take out commented code prior to submission, we'll start getting harder on this next week.